examples/handlers/simple_async_handlers_await/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "gotham_examples_handlers_simple_async_handlers_await"
+description = "An example that does asynchronous work before responding"
+version = "0.0.0"
+authors = ["David Laban <alsuren@gmail.com>"]
+publish = false
+edition = "2018"
+
+[dependencies]
+gotham = { path = "../../../gotham" }
+gotham_derive = { path = "../../../gotham_derive" }
+
+hyper = "0.13.1"
```

Comment:
Since https://github.com/gotham-rs/gotham/pull/393 is merged, let's use the re-exported hyper